gapis/api/vulkan/vulkan.api

```diff
@@ -227,6 +229,9 @@ sub ref!ExtensionSet supportedDeviceExtensions() {
   supported.ExtensionNames["VK_KHR_maintenance1"] = true
   supported.ExtensionNames["VK_KHR_maintenance2"] = true
   supported.ExtensionNames["VK_KHR_maintenance3"] = true
+  supported.ExtensionNames["VK_KHR_maintenance3"] = true
```

Comment:
This one is extra and duplicates the previous line.

Reply:
Done